src/Engine/ProtoCore/Core.cs

```diff
@@ -2116,10 +1865,20 @@ public void GenerateExprExeInstructions(int blockScope)
             }
         }
 
+        private RuntimeData GenerateRuntimeData()
+        {
+            Validity.Assert(RuntimeData != null);
+            RuntimeData.FunctionTable = FunctionTable;
```

Comment:
Ok, the intention was to create a new runtimedata and populate it. will follow this ip in next PR to make sure no regression happen when this is merged